src/metabase/routes/index.clj

```diff
@@ -66,21 +65,29 @@
   (load-template
    (str "frontend_client/" entrypoint-name ".html")
    (let [{:keys [anon_tracking_enabled google_auth_client_id], :as public-settings} (public-settings/public-settings)]
-     {:bootstrapJSON      (escape-script (json/generate-string public-settings))
+     {:bootstrapJS        (slurp (io/resource "frontend_client/index_inline_bootstrap.js"))
+      :googleAnalyticsJS  (slurp (io/resource "frontend_client/index_inline_ganalytics.js"))
+      :webFontConfigJS    (slurp (io/resource "frontend_client/index_inline_webfontconfig.js"))
+      :bootstrapJSON      (escape-script (json/generate-string public-settings))
       :localizationJSON   (escape-script (load-localization))
       :uri                (h.util/escape-html uri)
       :baseHref           (h.util/escape-html (base-href))
       :embedCode          (when embeddable? (embed/head uri))
       :enableGoogleAuth   (boolean google_auth_client_id)
       :enableAnonTracking (boolean anon_tracking_enabled)})))
 
+(defn- load-init-template []
+  (load-template
+    "frontend_client/init.html"
+    {:initJS (slurp (io/resource "frontend_client/init_inline.js"))}))
+
 (defn- entrypoint
   "Repsonse that serves up an entrypoint into the Metabase application, e.g. `index.html`."
   [entrypoint-name embeddable? {:keys [uri]} respond raise]
   (respond
    (-> (if (init-status/complete?)
          (resp/response (load-entrypoint-template entrypoint-name embeddable? uri))
-         (resp/resource-response "frontend_client/init.html"))
+         (resp/response (load-init-template)))
```

Comment:
Now that these are both calls to `resp/response` I would move the `if` to inside a single call e.g.

```clj
(resp/response (if (init-status/complete?)
                 (load-entrypoint-template entrypoint-name embeddable? uri)
                 (load-init-template)))
```

Reply:
Done.